docs/user/examples.rst

```diff
@@ -0,0 +1,350 @@
+Examples
+========
+
+Feature Management
+------------------
+
+Creating a feature type
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To begin, let's start by creating a new feature type in GeoMesa with the
+``create`` command. The ``create`` command takes three required and one
+optional flag:
+
+**Required**
+
+-  ``-c`` or ``--catalog``: the name of the catalog table
+-  ``-f`` or ``--feature-name``: the name of the feature
+-  ``-s`` or ``--spec``: the ``SimpleFeatureType`` specification
+
+**Optional**
+
+-  ``-dtg``: the default date attribute of the
+   ``SimpleFeatureType``
+
+Run the command:
+
+.. code:: bash
+
+    $ geomesa create -u <username> -p <password> \
+    -c cmd_tutorial \
+    -f feature \
+    -s id:String:index=true,dtg:Date,geom:Point:srid=4326 \
```

Comment:
in some cases id is a reserved word. i don't think we've documented all the places it will break things, though. in most cases it might not matter at all, but just in case...